perf/Microsoft.AspNetCore.Server.Kestrel.Performance/configs/CoreConfig.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using BenchmarkDotNet.Configs;
+using BenchmarkDotNet.Engines;
+using BenchmarkDotNet.Environments;
+using BenchmarkDotNet.Jobs;
+using BenchmarkDotNet.Validators;
+
+namespace Microsoft.AspNetCore.Server.Kestrel.Performance
+{
+    public class CoreConfig : ManualConfig
+    {
+        public CoreConfig()
+        {
+            Add(JitOptimizationsValidator.FailOnError);
+            Add(new RpsColumn());
+
+            Add(Job.Default.
+                With(Platform.X64).
+                With(Jit.RyuJit).
+                With(BenchmarkDotNet.Environments.Runtime.Core).
+                WithRemoveOutliers(true).
```

Comment:
I saw the following output when running the request parsing bencharks on my machine. Is this normal?

```
*** Hints ***
Outliers
  RequestParsing.ParsePlaintext: RemoveOutliers=True, Jit=RyuJit, Platform=X64, Runtime=Core, Server=True, LaunchCount=3, RunStrategy=Throughput, TargetCount=10, WarmupCount=5           -> Something went wrong with outliers: Size(MainTarget) = 30, Size(MainTarget/Outliers) = 1, Size(Result) = 30), RemoveOutliers = True
  RequestParsing.ParsePipelinedLiveAspNet: RemoveOutliers=True, Jit=RyuJit, Platform=X64, Runtime=Core, Server=True, LaunchCount=3, RunStrategy=Throughput, TargetCount=10, WarmupCount=5 -> 1 outlier  was  removed
  RequestParsing.ParseUnicodePipelined: RemoveOutliers=True, Jit=RyuJit, Platform=X64, Runtime=Core, Server=True, LaunchCount=3, RunStrategy=Throughput, TargetCount=10, WarmupCount=5    -> Something went wrong with outliers: Size(MainTarget) = 30, Size(MainTarget/Outliers) = 1, Size(Result) = 26), RemoveOutliers = True
```



`1 outlier  was  removed` seems OK, but I'm not sure about `Something went wrong with outliers: Size(MainTarget) = 30, Size(MainTarget/Outliers) = 1, Size(Result) = 26).`